app/ts/constants.ts

```diff
@@ -1,7 +1,4 @@
-export const MEV_RELAY_MAINNET = 'https://relay.dark.florist/'
-export const MEV_RELAY_GOERLI = 'https://relay-goerli.dark.florist/'
-
-export const BLOCK_EXPLORERS: { [chainId: string]: string } = {
-	'1': 'https://etherscan.io/',
-	'5': 'https://goerli.etherscan.io/'
-}
+export const NETWORKS: { [chainId: string]: { blockExplorer: string, mevRelay: string, rpcUrl: string } } = {
+	'1': { mevRelay: 'https://relay.dark.florist', blockExplorer: 'https://etherscan.io/', rpcUrl: 'https://rpc.dark.florist/flipcardtrustone' },
+	'5': { mevRelay: 'https://relay-goerli.dark.florist', blockExplorer: 'https://goerli.etherscan.io/', rpcUrl: 'https://rpc-goerli.dark.florist/flipcardtrustone' }
+} as const
```

Comment:
you can do without the definition in `NETWORKS` is cast with `const`
```suggestion
export const NETWORKS = {
	'1': { mevRelay: 'https://relay.dark.florist', blockExplorer: 'https://etherscan.io/', rpcUrl: 'https://rpc.dark.florist/flipcardtrustone' },
	'5': { mevRelay: 'https://relay-goerli.dark.florist', blockExplorer: 'https://goerli.etherscan.io/', rpcUrl: 'https://rpc-goerli.dark.florist/flipcardtrustone' }
} as const
```